akka-http-argonaut/src/main/scala/de/heikoseeberger/akkahttpargonaut/ArgonautSupport.scala

```diff
@@ -36,9 +38,11 @@ object ArgonautSupport extends ArgonautSupport
   */
 trait ArgonautSupport {
 
+  def unmarshallerContentTypes: Seq[ContentTypeRange] = Seq(`application/json`)
```

Comment:
Please add a line break after the signature (and equals sign) here and in the other files.